### DISCUSSION TITLE
How to change the navbar logo based on the current locale

### DISCUSSION THREAD

**Discussion by aliaz0** [facebook/docusaurus#8472] [Q&A]
How can I use different navbar logos for different locales? I have two logos for my app, an English version, and an Arabic version. I want to change the logo based on the active locale. how can I do it?

**Comment by slorber** [comment]
https://thisweekinreact.com/

<img width="412" alt="CleanShot 2022-12-21 at 19 32 31@2x" src="https://user-images.githubusercontent.com/749374/208978285-b9356c48-8201-4ade-b7d3-2cd262a0a411.png">

```js
i18n: {
    defaultLocale: "en",
    locales: ["en", "fr"],
    localeConfigs: {
      en: {
        label: "🇬🇧 English",
      },
      fr: {
        label: "🇫🇷 Français",
      },
    },
  },
```

Hope it helps?

If you want to use SVG/image logos I don't think we provide an easy way to do that atm, but we could make it easy to pass a "locale className" that you can target with custom CSS

**Reply by aliaz0** [reply]
Thanks for your reply @slorber , but I want to change the navbar main logo (not the locale flag) based on the current locale. For example, if the user changes the locale from 'en' to 'fr', see the different logo in the navbar.
Something like this:

`
themeConfig: {
    navbar: {
      title: 'My App',
      logo: {
        src: '${i18n.logoPath}/logo.png'
      }
   }
}
`

**Reply by slorber** [reply]
Then you need to swizzle LocaleDropdownNavbarItem and implement this on your own, we don't have a way to do that atm
